kyuubi-common/src/main/scala/org/apache/kyuubi/config/KyuubiConf.scala

```diff
@@ -254,6 +254,26 @@ object KyuubiConf {
   //                              Frontend Service Configuration                                 //
   /////////////////////////////////////////////////////////////////////////////////////////////////
 
+  object FrontendProtocols extends Enumeration {
+    type FrontendProtocol = Value
+    val THRIFT_BINARY, REST = Value
+  }
+
+  val FRONTEND_PROTOCOLS: ConfigEntry[Seq[String]] =
+    buildConf("frontend.protocols")
+      .doc("A comma separated list for all frontend protocols " +
+        "<ul>" +
+        " <li>THRIFT_BINARY - HiveServer2 compatible thrift binary protocol.</li>" +
+        " <li>REST - Kyuubi defined REST API(experimental).</li> " +
+        "</ul>")
```

Comment:
Shall we point out `THRIFT_BINARY ` is the default protocol? if users do nothing.